### PR TITLE
[Konvoy] Release Notes 2.1.5

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://d2iq.atlassian.net/browse/D2IQ-94232

## Description of changes being made

Release Notes 2.1.5

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4663.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by charIesdu — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
Needs to be copied to Kommander?

File: pages/dkp/konvoy/2.1/release-notes/2.1.5/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+## Fixes and Improvements
+
+### Wrong MachineDeployment (nodepool) names after Konvoy v1.8.5 to DKP v2.1.3 upgrade (D2IQ-93614)
+
+Corrected an issue where a `MachineDeployment` would be incorrectly named when running `dkp describe cluster` after upgrading from Konvoy v1.8. to v2.1.
+
+### Preprovisioned cluster adoption is broken (D2IQ-94144)
+
+Corrected an issue that prevented the adoption of pre-provisioned clusters in Konvoy 2.1.
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/release-notes/2.1.5/index.md:31]: @alejandroEsc  - Do we need to update some docs in this release somewhere to mention KIB 1.5.2?  As I believe this release requires that version, right?   

If the above is true, then:

@mmcelroy91  - I think we should also update your KIB table when this releases to mention that DKP 2.1.5 requires KIB 1.5.2

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.1/release-notes/2.1.5/index.md (hunk 1 above)
Comment (alejandroEsc) [reply pages/dkp/konvoy/2.1/release-notes/2.1.5/index.md:31]: yes, that is correct. The wording should say "prevented the upgrade"

File: pages/dkp/konvoy/2.1/release-notes/2.1.5/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+## Fixes and Improvements
+
+<!-- vale Vale.Avoid = NO -->
+### Wrong MachineDeployment (nodepool) names after Konvoy v1.8.5 to DKP v2.1.3 upgrade (D2IQ-93614)
+<!-- vale Vale.Avoid = YES -->
+
+Corrected an issue where a `MachineDeployment` would be incorrectly named when running `dkp describe cluster` after upgrading from Konvoy v1.8. to v2.1.
+
+<!-- vale Vale.Avoid = NO -->
+### Preprovisioned cluster adoption is broken (D2IQ-94144)
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/release-notes/2.1.5/index.md:32]: Consistency with the hyphens I guess?

```suggestion
### Pre-provisioned cluster adoption is broken (D2IQ-94144)
```

**Review by msdolbey — APPROVED** [review]
👍 
Wunderbar

**Comment by mwrightd2** [comment]
Added kommander notes, pushing live.

**Pull request merged**